src/Libraries/CoreNodeModels/Logic/If.cs

```diff
@@ -17,10 +17,15 @@ namespace CoreNodeModels.Logic
     public class If : NodeModel
     {
         [JsonConstructor]
-        private If(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts) { }
+        private If(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
```

Comment:
These changes basically add lacing capability to the If UI node however they are not quite related to this JIRA task.